src/trufflesom/interpreter/Primitive.java

```diff
@@ -82,4 +96,14 @@ public static void propagateLoopCount(final long count) {
       m.propagateLoopCountThroughoutLexicalScope(count);
     }
   }
+
+  @Override
+  public boolean isTrivial() {
+    return true;
```

Comment:
You mentioned "trivial" meaning elements can be inlined into the dispatch chain, is that all there is to it? I ask since I was curious about what that feature did exactly. Has to do with inlining in general, I assume

Reply:
It's not the same as inlining of blocks on the AST level.

`trivial` might be a bit of a misnomer, but the idea is that these methods don't do anything that prevents them from working when simply copied into the dispatch chain.
So, they implement the `PreevaluatedExpression` interface, and don't access anything unexpected. There are a few known patterns that I cover with this.

It includes getters, setters, returning of constants/literals, and now all primitives.